starter-code/functions-and-arrays.js

```diff
@@ -1,14 +1,25 @@
 // Find the maximum
 function maxOfTwoNumbers (first, second) {
-
+ if ( first < second) {
+   console.log(second);
+ } else {
+   console.log(first);
```

Comment:
You are showing the result in the console instead of returning it. That way, the variable 'largest' in line 10 is going to be undefined (because you are not returning anything). Usually you want your functions to return something, that way you can use that value ahead.